src/quo/components/links/internal_link_card/channel/view.cljs

```diff
@@ -0,0 +1,92 @@
+(ns quo.components.links.internal-link-card.channel.view
+  (:require
+    [quo.components.icon :as icon]
+    [quo.components.links.internal-link-card.channel.style :as status-link-previews-style]
+    [quo.components.links.link-preview.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as theme]
```

Comment:
require `[quo.theme :as theme] as `[quo.theme]` to avoid `theme` clashing with prop